script.js

```diff
@@ -0,0 +1,108 @@
+const inputText = document.querySelector('#todo-input');
+const addTodoBtn = document.querySelector('#add-todo-btn');
+const todoListTitle = document.querySelector('#todo-list-title');
+const doneListTitle = document.querySelector('#done-list-title');
+const todoList = document.querySelector('.todo-list');
+const doneList = document.querySelector('.done-list');
+
+let items = [];
+
+// 입력한 할 일을 items에 추가
+const addNewTodo = () => {
+  const todoObject = {
+    id: null,
+    text: inputText.value,
+    isDone: false,
+  };
+
+  // 공백 입력, 중복 입력 방지
+  const index = items.findIndex((todo) => todo.text == inputText.value);
+  if (inputText.value && index === -1) {
+    items.push(todoObject);
+    render();
+  }
+
+  // 할 일 추가 후 인풋 값 비우기
+  inputText.value = '';
+};
+
+addTodoBtn.addEventListener('click', addNewTodo);
+
+// 엔터를 눌러도 입력되도록
+const enterKey = () => {
+  if (window.event.keyCode == 13) {
+    addNewTodo();
+  }
+};
+
+// 삭제 버튼을 누르면 해당하는 항목 삭제
+const deleteTodo = (e) => {
+  items = items.filter((todo) => 'bin' + todo.id !== e.target.id);
+  render();
+};
+
+// 할 일의 isDone을 토글
+const toggleTodo = (e) => {
+  const todo = items.find((todo) => 'todo' + todo.id === e.target.id);
+  todo.isDone = !todo.isDone;
+
+  render();
+};
+
+// 할 일과 완료한 일의 개수 세기
+const countTodo = () => {
+  const doneTodoCnt = items.filter((todo) => todo.isDone).length;
+  const todoCnt = items.length - doneTodoCnt;
+  todoListTitle.innerHTML = `📋 TO DO (${todoCnt})`;
+  doneListTitle.innerHTML = `💿 DONE (${doneTodoCnt})`;
+};
```

Comment:
이 함수는 따로 함수로 만들기보다 그냥 render 내부에 있는 코드로만 있는건 어떨까 싶습니다!
```suggestion
const renderTodoCount = () => {
  const doneTodoCnt = items.filter((todo) => todo.isDone).length;
  const todoCnt = items.length - doneTodoCnt;
  todoListTitle.innerHTML = `📋 TO DO (${todoCnt})`;
  doneListTitle.innerHTML = `💿 DONE (${doneTodoCnt})`;
};
```

함수의 이름도 함수가 하는 일을 더 잘 반영하도록 작성해보면 좋을 것 같습니다

Reply:
처음에 렌더 함수 안에 작성했다가 로컬스토리지가 비어 있을 때 저 카운터 자체가 불러와지지 않아서 따로 분리한 거였는데, 그냥 렌더함수 안에 작성하고 렌더함수를 호출하면 되는 거였네요...... 다시 수정했습니다 짚어 주셔서 감사합니다!